### ISSUE TITLE
Tried to change to this version of the plugin but auto-format was not working

### ISSUE THREAD

**Issue by mpmont** [driade/phpfmt-8#6]
Basically I uninstalled the other plugin. Installed this one by cloning the repo into the packages folder. 

Added the user config file.

Changed the path to php 8.

Format on save was not working and there was no errors in the console.

**Comment by driade** [comment]
Maybe you should restart Sublime Text. Do you see the phpfmt8 options in the command palette?

**Comment by mpmont** [comment]
Yes I did see the options in the command palette. I'm try and add the plugin again later today and see if it works with the correct folder name.
